static/beta/itless/navigation/rhel-navigation.json

```diff
@@ -413,14 +413,20 @@
           "subtitle": "Red Hat Insights for RHEL",
           "description": "Perform Red Hat-recommended analysis across your Red Hat Enterprise Linux fleet, like upgrade and conversion readiness.",
           "alt_title": [
-              "Tasks",
-              "task",
-              "CentOS conversion",
-              "Pre Conversion analysis",
-              "Pre-Conversion analysis",
-              "Convert to RHEL",
-              "C2R",
-              "CentOS"
+            "Tasks",
+            "tasks""task",
```

Comment:
```suggestion
            "tasks", "task",
```

Reply:
And I see many other places with the same issue (won't be commenting on them)

Reply:
yup, my slopy copy-pasta